### DISCUSSION TITLE
Contract is not Verifyed like in the course , how to fix?

### DISCUSSION THREAD

**Discussion by Abdo-al-R** [Cyfrin/foundry-full-course-cu#2578] [Q&A] [answered]
Contract is not verifyed like in the course , how to fix?

course :

![Screenshot_4](https://github.com/user-attachments/assets/9bc4d653-1818-44c5-b0d1-8ae9229074cf)
![Screenshot_3](https://github.com/user-attachments/assets/bbed2104-2269-4730-aa94-299d496c4479)

mine looks like it is deployed but the code not showing on zksync explorer cuz its not verifyed :

![Screenshot_1](https://github.com/user-attachments/assets/63564aaf-c184-47a9-b430-4a4b7c689d08)
![Screenshot_2](https://github.com/user-attachments/assets/854e2683-e3a0-4102-9ad9-8c7b59058057)

**Comment by EngrPips** [comment, marked as answer]
Hello @Abdo-al-R, verification on ZKSync is not working smoothly as of now. Please ignore it if it doesn't work for you.

**Reply by Abdo-al-R** [reply]
Alright thank you

**Reply by EngrPips** [reply]
Anytime, Friend.
